lib/portfolio_page/pie_chart.dart

```diff
@@ -0,0 +1,272 @@
+import 'package:equity_gals_portfolio/portfolio_page/portfolio_page.dart';
+import 'portfolio_page.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:flutter/material.dart';
+
+class EGPieChart extends StatefulWidget {
+  final double maxContentWidth;
```

Comment:
You passed this value into the constructor of the EGPPieChart but it is not used and also it doesn't respond amazingly to a small screen, like mobile.
![Uploading image.png…]()